src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -141,22 +141,22 @@ public function assertMissing($path)
 
         foreach ($paths as $path) {
             PHPUnit::assertFalse(
-                $this->exists($path), "Found unexpected file at path [{$path}]."
+                $this->exists($path), "Found unexpected file or directory at path [{$path}]."
             );
         }
 
         return $this;
     }
 
     /**
-     * Determine if a file exists.
+     * Determine if a file or directory exists.
      *
      * @param  string  $path
      * @return bool
      */
     public function exists($path)
     {
-        return $this->driver->fileExists($path);
+        return $this->driver->has($path);
```

Comment:
This was how it's currently on 8.x:

https://github.com/laravel/framework/blob/8.x/src/Illuminate/Filesystem/FilesystemAdapter.php#L127